### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Disk_DiskInfoTable_ScreenReader:Jaws does not read the table properly as the table rows and column reference and correct no. of rows present

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#2094]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;#MAS4.2.3 #JAWS; #Win32; 

Environment Details:
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
JAWS 2019
Windows10

Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and  click on it.
6. Navigate to Table under the tool bar.
7. Open JAWS and check whether it reads the table properly as the table rows and column reference and correct no. of rows present.

**​Actual:​**
Jaws does not read the table properly as the table rows and column reference and correct no. of rows present.
**Same issue is observed throughout the application wherever table present.**
**Expected:**
Jaws should read the table properly as the table rows and column reference and correct no. of rows present.
**UserImpact:**
Screen reader users unable to get the data present on the table correctly.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT Verified on version 1.12.0 and Jaws 2019.1912.1.The issue is still reproduced. Jaws does not read the correct row and column reference of the table when focus goes to the table.

**Comment by craxal** [comment]
@nirmalsahoo JAWS appears to be reading what we expect from my end, as does NVDA, so I'm not sure what the issue is. Can you share exactly what JAWS is reading and what you expect JAWS should say?

**Comment by nirmalsahoo** [comment]
@craxal We verified the above issue:
**Actual**issue is: As we are navigating on the Table with JAWS, JAWS does not announce column header name with table cells.
**For Ex**: For "ally" table cell column header is "Disk name" but here screen reader not announces column header name along with table cell instead JAWS announces entire row content in single attempt.

**Expected**: JAWS should announce column header name, row number and also column number with table cells. And JAWS should not announce entire row content in single attempt when we navigate to Table cells it should announce table cells individually.
**For Ex**: For "ally" table cell column header is "Disk name", row is 2 and column is 1 here screen reader should announce like: "Ally Column header "Disk name" Row2 of 2 ,Column1 of 7   when we navigate to Table cells and it should announce like same information for all table cells individually.

[2094_A11y_AzureToolsStorageExplorer_Disk_DiskInfoTable_ScreenReader.zip](https://github.com/microsoft/AzureStorageExplorer/files/4280839/2094_A11y_AzureToolsStorageExplorer_Disk_DiskInfoTable_ScreenReader.zip)

**Comment by nirmalsahoo** [comment]
Verified the issue and is no longer reproduced.Hence closing this issue.Please refer the attachment 2094_A11y_AzureToolsStorageExplorer_Disk_DiskInfoTable_ScreenReader_Closed in [Onedrive](https://microsoft-my.sharepoint.com/:f:/p/v-nisaho/EsSUKgfoOK9HmEL8PqqXd94BUtAVBzKJetpyH6pKAsUxyw?e=bAKDhH) link

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
